pkg/odo/cli/cli.go

```diff
@@ -41,13 +42,16 @@ var (
  /  \__/    Find more information at https://odo.dev
  \__/`
 
-	odoExample = ktemplates.Examples(`  # Creating and deploying a Node.js project
-  git clone https://github.com/odo-devfiles/nodejs-ex && cd nodejs-ex
-  %[1]s create nodejs
-  %[1]s push
+	odoExample = ktemplates.Examples(`Initialize and take your pick from multiple languages or frameworks:
+  %[1]s init
+
+	After creating your initial application, start development with:
+  %[1]s dev
+
+	Want to deploy to production? See it live with:
```

Comment:
I am quite sure we don't want to promote odo as a tool that deploys anything to prod. Besides, it also hints that the things that were created as a part of `odo dev` would magically work with `odo deploy`. I understand that need to highlight `odo deploy`, but the wording here is going to set expectations odo can't fulfill (because they are non-goals).